### PR TITLE
Move entropy/ulimit docs to troubleshooting

### PR DESCRIPTION
**Pull request by aledsage**
They were previously in website/documentation/increase-entropy.md and
in the faq. Now they are top-level pages in troubleshooting.

### CODE REVIEW

File: guide/ops/troubleshooting/increase-system-resource-limits.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: website-normal
+title: Increase System Resource Limits
+toc: /guide/toc.json
+---
+
+You could encounter the error below when running with many entities:
```
Comment (drigodwin) [inline guide/ops/troubleshooting/increase-system-resource-limits.md]: I would rephrase to "If you encounter the error..." or "You may encounter the error..."

**Comment by drigodwin** [comment]
Looks good! thanks @aledsage

**Comment by aledsage** [comment]
Thanks @drigodwin - I've made that change. Merging now.
